var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -48,6 +48,7 @@ class Elpa(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     variant("openmp", default=True, description="Activates OpenMP support")
     variant("mpi", default=True, description="Activates MPI support")
+    variant("gpu_streams", default=True, description="Enable GPU streams support")
```

Comment:
Optional:
```suggestion
    variant("gpu_streams", default=True, description="Activates GPU streams support")
```

Reply:
Also, is a version constraint necessary for this variant?

Reply:
No version constraint is necessary, the first version included in Spack already supports GPU streams (as opt-in).